client/reader/discover/controller.js

```diff
@@ -70,11 +86,25 @@ const exported = {
 				/>
 			</>
 		);
-		/* eslint-enable wpcalypso/jsx-classname-namespace */
+		next();
+	},
+
+	discoverSsr( context, next ) {
+		context.renderHeaderSection = renderHeaderSection;
+
+		const selectedTab = context.query.selectedTab || DEFAULT_TAB;
+		const tabTitle = getSelectedTabTitle( selectedTab );
+		context.primary = (
+			<>
+				<DiscoverPageDocumentHead tabTitle={ tabTitle } />
+				<DiscoverHeader selectedTab={ selectedTab } />
+				<PostPlaceholder />
+			</>
+		);
 		next();
 	},
 };
 
 export default exported;
```

Comment:
Could we get rid of this `exported` object and simply export the `discover*` functions as plain ESM exports? The object is probably some remnant of old CommonJS modules.